lib/dock.rb

```diff
@@ -0,0 +1,34 @@
+class Dock
+  attr_reader :name, :max_rental_time, :rental_time
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @revenue_multiplier = []
```

Comment:
Instead of holding on to the `price-per_hour`, why not hold on to all the rental information (renter and boats)?